hack/jenkins/e2e.sh

```diff
@@ -649,12 +649,15 @@ case ${JOB_NAME} in
     : ${E2E_DOWN:="true"}
     : ${E2E_TEST:="false"}
     : ${USE_KUBEMARK:="true"}
-    : ${NUM_MINIONS:="40"}
-    : ${E2E_ZONE:="asia-east1-a"}
-    : ${MASTER_SIZE:="n1-standard-32"}  # Note: not available in all zones
-    : ${MINION_SIZE="n1-standard-16"}   # Note: can fit about 17 hollow nodes per core
+    : ${NUM_MINIONS:="8"}
+    : ${MASTER_SIZE:="n1-standard-4"}
+    : ${MINION_SIZE="n1-standard-8"}   # Note: can fit about 17 hollow nodes per core
     #                                           so NUM_MINIONS x cores_per_minion should
     #                                           be set accordingly.
+    # Override defaults and set kubemark parameters
+    E2E_ZONE="asia-east1-a"
+    KUBEMARK_MASTER_SIZE="n1-standard-16"
+    KUBEMARK_NUM_MINIONS="1000"
```

Comment:
Can you please add comments to both kubemark suite what do we expect them to test (i.e. size of the kubemark cluster and size of the underlying cluster)? That would be very helpful.